viewer/viewer.js

```diff
@@ -8068,6 +8068,45 @@ function updateHuntStatus (req, res, status, successText, errorText) {
   });
 }
 
+function validateUserIds (userIdList) {
+  return new Promise((resolve, reject) => {
+    const query = {
+      _source: ['userId'],
+      from: 0,
+      size: 10000,
+      query: { // exclude the shared user from results
+        bool: { must_not: { term: { userId: '_moloch_shared' } } }
+      }
+    };
+
+    Db.searchUsers(query, (error, users) => {
+      if (error) {
+        reject('Unable to validate userIds');
+      }
+
+      let usersList = [];
+      for (let user of users.hits.hits) {
+        usersList.push(user._source.userId);
+      }
+
+      let validUsers = [];
```

Comment:
I think youngins would use sets and filter here :)

Reply:
:roll_eyes: